solr/core/src/java/org/apache/solr/cloud/api/collections/BackupCmd.java

```diff
@@ -116,6 +116,10 @@ public void call(ClusterState state, ZkNodeProps message, NamedList<Object> resu
           break;
         }
         case CollectionAdminParams.NO_INDEX_BACKUP_STRATEGY: {
+          NamedList<Object> response = new SimpleOrderedMap<>();
+          response.add("collection", collectionName);
+          response.add("startTime", backupProperties.getStartTime());
+          results.add("response", response);
```

Comment:
Good catch! I have totally forgot about this strategy %(

Reply:
Me too, luckily there was a test that caught it!